gemfiles/common.rb

```diff
@@ -1,3 +1,11 @@
 source 'https://rubygems.org'
 
 gemspec path: Bundler.root.sub('/gemfiles', '')
+
+gem 'minitest'
+gem 'minitest-rg'
```

Comment:
FYI [maxitest](https://github.com/grosser/maxitest) is a nice gem that includes rg and a few other plugins